### ISSUE TITLE
User withdraws money

### ISSUE THREAD

**Issue by aknanayaw** [aknanayaw/Init#1]
As a Grey Mobile Money User  
I want to withdraw money  
So that I can buy things

### Scenario 1: 
Given the Mobile Money User's mobile phone number is in the mobile money database  
When the Mobile Money User's details are submitted to the system  
Then the system generates an OTP

### Scenario 2: 
Given the Mobile Money User's mobile phone number is NOT in the mobile money database  
When the Mobile Money User's details are submitted to the system  
Then the system generates an error message

**Comment by ramedalor** [comment]
Exactly. But the error can be that the person should check the details again

**Comment by PrinceBobby** [comment]
## Scenario 3:##

Given the Mobile Money User's mobile phone number is in the Mobile Money database but the User has insufficient Balance in their Mobile Money wallet.
When the Mobile Money User's details are submitted to the system.
The system generates a prompt notifying the user of insufficient balance and the amount left in the User's account.
